components/vf-card-container/vf-card-container.config.yml

```diff
@@ -21,22 +21,20 @@ context:
     - To promote molecular biology across Europe
     - To create a centre of excellence for Europe's leading young molecular biologists
   vf_cards:
-    - card_type: easy
-      card_title: One card
+    - card_title: One card
```

Comment:
did you mean to also update vf-card-container?

Reply:
yikes - not in this one

Reply:
I'll close this and start afresh